src/extension/react/components/Panel.tsx

```diff
@@ -29,9 +30,13 @@ ModuleRegistry.registerModules([
   CellStyleModule,
   ScrollApiModule,
   TooltipModule,
+  RowApiModule,
 ]);
 import { useDevtronContext } from '../context/context';
 
+type UUID = string;
+type SerialNumber = number;
```

Comment:
[non-blocking] Move these up to the top-level shared types file to tie these types with the IPCEventData shape more explicitly
```suggestion
// in src/types/shared.ts
export type UUID = string;
export type SerialNumber = number;
...
export interface IpcEventData {
    ....
    uuid?: UUID,
}
...
export interface IpcEventDataIndexed extends IpcEventData {
  serialNumber: SerialNumber;
  gotoSerialNumber?: SerialNumber; // For navigating to a specific event in the grid
}
```

Reply:
Fixed, this was batched in [7d5813e](https://github.com/electron-userland/devtron/pull/275/commits/7d5813e5d632d4f7883c46f06add3a3ac3f08a79).